### PR TITLE
added a file 

### PR DESCRIPTION
**Pull request by dikeshnetam96**
one test class to check something

### CODE REVIEW

File: src/main/scala/assessment/TestClass.scala [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package assessment
+
+object TestClass extends App{
+  val num = 7
+  val str = "string"
```
Comment (DKNetam123) [inline src/main/scala/assessment/TestClass.scala]: change this variable to double

File: src/main/scala/assessment/TestClass.scala [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+  println("Hii")
```
Comment (DKNetam123) [inline src/main/scala/assessment/TestClass.scala]: change it to bye....

**Review by DKNetam123 — CHANGES_REQUESTED** [review]
please do it asap

File: src/main/scala/assessment/TestClass.scala (hunk 2 above)
Comment (dikeshnetam96) [reply src/main/scala/assessment/TestClass.scala]: updated

**Review by DKNetam123 — APPROVED** [review]

**Pull request merged**